src/deluge/model/drum/midi_drum.cpp

```diff
@@ -113,7 +113,7 @@ int8_t MIDIDrum::modEncoderAction(ModelStackWithThreeMainThings* modelStack, int
 
 	NonAudioDrum::modEncoderAction(modelStack, offset, whichModEncoder);
 
-	if ((getCurrentUI() == &instrumentClipView || getCurrentUI() == &automationClipView)
+	if ((getCurrentUI() == &instrumentClipView || getCurrentUI() == &automationView)
```

Comment:
can this have a problem if in arranger automation view? I think we can get here via midi while the clip isn't open

Reply:
No because this is modEncoderAction which refers to the use of the deluge's mod encoders. I believe this function is for MIDI in a kit clip?

Also the UI mode UI_MODE_AUDITIONING is not used in arranger view.